content/en/docs/components/pipelines/v2/version-compatibility.md

```diff
@@ -0,0 +1,27 @@
++++
+title = "Version Compatibility"
+description = "Version compatibility between KFP Runtime and KFP SDK"
+weight = 80
++++
+
+{{% kfp-v2-keywords %}}
+
+The following table presents a comprehensive overview of the version compatibility between Kubeflow Pipelines (KFP) Runtime and KFP SDK.
+
+| KFP Runtime | KFP SDK | Notes |
+|---|---|---|
+| v2.0.* | v2.0.* | Active development. Support for certain features may be staged between the Runtime and the SDK. |
+| v2.0.* | v1.8.* | Backward compatibility maintained. No v2 features support. |
```

Comment:
It's not entirely clear to me what is meant by backward compatibility here. It seems it could be within the KFP Runtime and KFP SDK individually, between them, or something else. Is there a way we can clarify this a bit?

Also, can you help me understand what is meant by "No v2 features support." This is confusing to me since features from the v1.8 SDK's `kfp.v2` namespace are supported by the v2.0 backend. I think I'm thinking about this the wrong way and perhaps other readers will too.

Reply:
Added some clarification notes, see if it helps. 

> This is confusing to me since features from the v1.8 SDK's kfp.v2 namespace are supported by the v2.0 backend.

They're not officially supported by v2.0 backend. Specifically, If they were compiled by v2 compiler, they would be IR in JSON format, and we disallow JSON in v2.0 backend. If they were compiled by v1 compiler using v2-compatible mode, they are Argo in YAML format, which is technically working on either v1 or v2 backend. But we have declared end of support for v2-compatible mode a long time ago. 
To avoid further confusing the users with all these details, I think simply put "they're not supported" is sufficient. WDYT?